multiversx_sdk_wallet/user_wallet.py

```diff
@@ -54,7 +54,10 @@ def from_mnemonic(cls, mnemonic: str, password: str, randomness: Union[IRandomne
 
     @classmethod
     def decrypt_secret_key(cls, keyfile_object: Dict[str, Any], password: str) -> UserSecretKey:
-        # Here, we do not check the "kind" field. Older keystore files (holding only secret keys) do not have this field.
+        # Here, we check the "kind" field only for files that have it. Older keystore files (holding only secret keys) do not have this field.
```

Comment:
We should apply this fix in JavaScript, as well:

https://github.com/multiversx/mx-sdk-js-wallet/blob/main/src/userWallet.ts#L98

Reply:
Will do.